libraries/Microsoft.Bot.Builder.Core.Extensions/ActivityFilterMiddleware.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Core.Extensions
+{
+    public class ActivityFilterMiddleware : IMiddleware
```

Comment:
>ActivityFilterMiddleware [](start = 17, length = 24)

Then I would just rename this to FilterMiddleware

Reply:
@tomlm Based on the comments above, I do think it is the naming that is throwing things off.  Maybe it should be renamed to something like HandleActivityTypeMiddleware?